app/routes/workspace-layout.tsx

```diff
@@ -54,9 +54,11 @@ export default function Index() {
           <Flex alignItems='center' gap={1} style={{ paddingInline: 'var(--size-2)', marginInlineStart: 'auto', flex: '0 0 auto' }}>
             <EditorsControl />
             <Separator orientation='vertical' style={{ margin: 'var(--size-2)' }} />
+            <Field direction='row' alignItems='center' gap={2} title={t('browser.openSouthView')} aria-label={t('browser.openSouthView')}>
```

Comment:
Do you know what a field is for?
It is to link a label to a input field. Neither is in use here, so it is not needed. 
Move the title/aria-label to the button direclty.